ui/component/autoplayCountdown/view.jsx

```diff
@@ -5,6 +5,10 @@ import UriIndicator from 'component/uriIndicator';
 import I18nMessage from 'component/i18nMessage';
 import { formatLbryUrlForWeb } from 'util/url';
 import { withRouter } from 'react-router';
+import debounce from 'util/debounce';
+
+const DEBOUNCE_SCROLL_HANDLER_MS = 150;
+const CLASSNAME_AUTOPLAY_COUNTDOWN = '.autoplay-countdown';
```

Comment:
This should just be 
`const CLASSNAME_AUTOPLAY_COUNTDOWN = 'autoplay-countdown';`

Then below on line 104, it can be changed to
`<div className={CLASSNAME_AUTOPLAY_COUNTDOWN}>`

Reply:
Ah, ok.  Re-did the "cleanup" commit.